rfs/root.go

```diff
@@ -256,10 +300,10 @@ func (r *rfsRoot) consultMirror(req *protobuf.Request) (permitted bool) {
 // idFromStat is a rip-off from go-fuse's loopback.go.
 func (r *rfsRoot) idFromStat(st *syscall.Stat_t) fs.StableAttr {
 	dev := r.LoopbackNode.RootData.Dev
-	swapped := (uint64(st.Dev) << 32) | (uint64(st.Dev) >> 32)
+	swapped := (st.Dev << 32) | (st.Dev >> 32)
```

Comment:
Good catch by the way. I didn't notice this redundant conversion. We may want to make a pull request to go-fuse as I literally stole this code from `loopback.go`.